provider/cmd/run.go

```diff
@@ -290,6 +293,11 @@ func doRunCmd(ctx context.Context, cmd *cobra.Command, _ []string) error {
 
 	gwaddr := viper.GetString(FlagGatewayListenAddress)
 
+	cert, err := cutils.LoadCertificateFromFrom(cctx.HomeDir, cctx.FromAddress, txFactory.Keybase())
```

Comment:
This seems like it is named weird ? It ends with "FromFrom" ?

Reply:
Changed to `LoadPEMForAccount`